frontend/api/sublet/offersApi.ts

```diff
@@ -0,0 +1,39 @@
+// /api/offersApi.ts
+import { CreateOfferRequest, OfferResponse } from '@/interfaces/sublet/Offer'
+import { doApiRequest } from '@/utils/fetch'
+
+const BASE_URL = 'api/sublet'
+
+// Get all offers for a specific sublet
+export const getOffers = async (subletId: string): Promise<OfferResponse[]> => {
+  const response = await doApiRequest(`${BASE_URL}/listOffers/${subletId}`)
+  if (!response.ok) {
+    throw new Error('Failed to fetch offers')
+  }
+  return response.json()
+}
+
+// Create a new offer for a sublet
+export const createOffer = async (
+  subletId: string,
+  data: CreateOfferRequest
+): Promise<void> => {
```

Comment:
Probably the response shouldn't be void? We should return the response so we can check it and maybe display an error or sucess message based off of it. 

In general is it nescecary to throw after each call (i have no preference here but just wondering). Why not just return the response and have the caller inspect it? It seems natural that for most responses we might want to handle the happy case and the bad case